src/nifreeze/cli/parser.py

```diff
@@ -29,13 +29,13 @@
 import yaml
 
 
-def _parse_yaml_config(file_path: Path) -> dict:
+def _parse_yaml_config(file_path: str) -> dict:
```

Comment:
```suggestion
def _parse_yaml_config(file_path: str | Path) -> dict:
```

Reply:
I do think the `Path` case will be ever hit: https://github.com/nipreps/nifreeze/commit/0a313000bf2ac35fce040055f86cc38a153854d7